build/IPDK_Container/Ubuntu20.04/scripts/get_p4ovs_repo.sh

```diff
@@ -0,0 +1,20 @@
+#Copyright (C) 2021 Intel Corporation
```

Comment:
Better to reuse the existing script as it is common procedure for both FC and Ubuntu distribution containers..

Reply:
Removed from Ubuntu20.04 folder.